modules/rpc/cookie.js

```diff
@@ -106,7 +106,6 @@ function getAuth(options) {
     auth = fs.readFileSync(COOKIE_FILE, 'utf8').trim();
   } else {
     auth = undefined;
-    log.debug('could not find cookie file! path:', COOKIE_FILE);
```

Comment:
we should let this, it's a good debug output that should not be displayed in release as logging levels will have been adjusted

Reply:
I'll re-insert it, it was annoying me